.storybook/preview.js

```diff
@@ -14,6 +14,15 @@ addDecorator((storyFn) => (
 ));
 
 export const parameters = {
+  viewMode: 'docs',
```

Comment:
This is the magic to make the docs view the default. That is very desirable so I push this change together with this PR to save me from extracting those changes 🙏